### PR TITLE
Update retry

### PR DESCRIPTION
**Pull request by jbrown-xentity**
So, according to [documentation](https://httpd.apache.org/docs/2.4/howto/reverse_proxy.html), it looks like we should be specific in proxying to localhost instead of `0.0.0.0`. Also, we noticed [timeout issues when the app was starting up](https://github.com/GSA/datagov-deploy/issues/2030), causing apache2 to wait for 60 seconds. This moves that down to 10 seconds, setting described [here](https://stackoverflow.com/questions/23709832/ap-proxy-connect-backend-disabling-worker-for-127-0-0-1).

@adborden thoughts on the specific timing, do we want 10, 30, 0?

### CODE REVIEW

**Review by adborden — APPROVED** [review]
👍  can you also look at the gunicorn configuration? Similarly, gunicorn should only be listening on localhost:5000, and not be accessible to other hosts (everything goes through apache).

File: ansible/roles/software/ckan/inventory/templates/etc/apache2/sites-enabled/ckan.conf [hunk 1]
```diff
@@ -10,8 +10,8 @@ WSGISocketPrefix /var/run/wsgi
 
     #gunicorn configuration
     ProxyPreserveHost On
-    ProxyPass / http://0.0.0.0:5000/
-    ProxyPassReverse / http://0.0.0.0:5000/
+    ProxyPass / http://127.0.0.1:5000/ retry=10
```
Comment (adborden) [inline ansible/roles/software/ckan/inventory/templates/etc/apache2/sites-enabled/ckan.conf]: Hmm, you could even make this as low as 1 second 🤷

**Comment by jbrown-xentity** [comment]
Added, will merge pending test completion.

File: ansible/roles/software/ckan/inventory/templates/etc/supervisor/conf.d/gunicorn-web-app.conf [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [program:gunicorn-web-app]
-command={{ project_source_new_code_path }}/bin/gunicorn --worker-class gevent --paste /etc/ckan/production.ini -b localhost:5000
+command={{ project_source_new_code_path }}/bin/gunicorn --worker-class gevent --paste /etc/ckan/production.ini -b localhost:5000 --proxy-allow-from="127.0.0.1"
```
Comment (adborden) [inline ansible/roles/software/ckan/inventory/templates/etc/supervisor/conf.d/gunicorn-web-app.conf:2]: fwiw, I think 127.0.0.1 is the [default](https://docs.gunicorn.org/en/stable/settings.html#proxy-allow-ips)

**Pull request merged**